collections/nemo_nlp/nemo_nlp/data/datasets/token_classification.py

```diff
@@ -41,20 +41,29 @@ def get_features(queries,
                  pad_label='O',
                  raw_labels=None,
                  unique_labels=None,
+                 label_ids=None,
                  ignore_extra_tokens=False,
                  ignore_start_end=False):
     """
     Args:
-        queries (list of str):
-        max_seq_length (int): max sequence length minus 2 for [CLS] and [SEP]
-        tokenizer (Tokenizer): such as NemoBertTokenizer
-        pad_label (str): pad value use for labels.
-            by default, it's the neutral label.
-        raw_labels (list of str): list of labels for every work in sequence
-        unique_labels (set): set of all labels available in the data
-        ignore_extra_tokens (bool): whether to ignore extra tokens in
+    queries (list of str): text sequences
+    max_seq_length (int): max sequence length minus 2 for [CLS] and [SEP]
+    tokenizer (Tokenizer): such as NemoBertTokenizer
+    pad_label (str): pad value use for labels.
+        by default, it's the neutral label.
+    raw_labels (list of str): list of labels for every word in a sequence
+    unique_labels (set): set of all labels available in the data
+        (required if raw_labels is not None and label_ids is None,
+        unique_labels are required for train set, but not for dev set)
```

Comment:
this requirement should be checked by code

Reply:
done